src/features/call/components/ParticipantMedia.module.css

```
@@ -26,20 +26,13 @@
-.screenShare .media {
-  object-fit: contain;
-}
-
 .selfPreview {
   position: absolute;
   top: var(--gap);
   left: var(--gap);
 
-  width: clamp(120px, 20%, 300px);
-  height: clamp(120px, 20%, 300px);
+  width: clamp(120px, 25%, 300px);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)